backend/TaskTree/Controllers/ProjectsController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using TaskTree.Models;
+using TaskTree.Models.Requests;
+using TaskTree.Models.Responses;
+
+namespace TaskTree.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProjectsController : ControllerBase
```

Comment:
All these controller actions should have `[Authorized]` tag and verify that the user modifying is the one that owns in (check `UsersController.cs` for reference)